### PR TITLE
Updated regex for HWG STE2 r2 to better detect hardware and software version

### PR DESCRIPTION
**Pull request by luc-ass**
This is a simple fix for the inaccurate regex. It now correctly displays hardware and software version for the STE2 r2

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] ~~If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.~~
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Could you also generate the json test data?

**Comment by luc-ass** [comment]
@Jellyfrog, I just noticed, that the initial discovery works but it reverts on automatic rediscovery. Do you have any idea what part of the discovery I might have missed?

File: includes/definitions/discovery/hwg-ste2.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 mib: STE2-MIB
 modules:
     os:
-        sysDescr_regex: '/(?<hardware>.+?)( (?<version>[\d.]+))?/'
+        sysDescr_regex: '/(?<hardware>.*),\s+fw:(?<version>.*)?/'
```
Comment (PipoCanaja) [inline includes/definitions/discovery/hwg-ste2.yaml:4]: not sure of this regexp (final '?') ?

You may try this : 
`sysDescr_regex: '/(?<hardware>[^,]*),\s+fw:(?<version>.*)/'`

Everything up to the "," is hardware, everything after "fw:" is version

Comment (PipoCanaja) [reply includes/definitions/discovery/hwg-ste2.yaml:4]: at least it seems to work as expected.

**Comment by PipoCanaja** [comment]
Let us know if the PR is ready from your side. From my side, looks good.

**Comment by luc-ass** [comment]
LGTM 🙏

**Review by PipoCanaja — APPROVED** [review]

**Pull request merged**